examples/plotting/file/slider_pycalback.py

```diff
@@ -0,0 +1,53 @@
+"""
```

Comment:
missing `l` in `pycallback` but I think a better name might be more like: `python_callback.py` thats the most important aspect to emphasize, the slider part is just a detail.